src/main/java/sg/ncl/testbed_interface/RestUtil.java

```diff
@@ -0,0 +1,15 @@
+package sg.ncl.testbed_interface;
+
+/**
+ * @author Te Ye
+ */
+import org.springframework.http.HttpStatus;
+
+public class RestUtil {
+
+    public static boolean isError(HttpStatus status) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)